packages/api/src/external/carequality/process-outbound-patient-discovery-resps.ts

```diff
@@ -45,11 +54,11 @@ export async function processOutboundPatientDiscoveryResps({
       results
     );
 
-    await processPatientDiscoveryProgress({ patient: patientIds, status: "completed" });
-
     const patient = await getPatientOrFail({ id: patientId, cxId });
-    const startedAt = patient.data.patientDiscovery?.startedAt;
+    const cqData = getCQData(patient.data.externalData);
 
+    // TODO Pass explicitly
+    const startedAt = cqData?.discoveryStartedAt;
```

Comment:
not to be picky, but what does this TODO mean? Are we leaving those or are you planning to tackling them before merging the PR?

Reply:
Removed. Not going to happen in this PR.